functions/Remove-DbaDatabaseSafely.ps1

```diff
@@ -384,7 +384,7 @@ function Remove-DbaDatabaseSafely {
 				}
 			}
 			catch {
-				Write-Message -Level Error -Message "Restore failed: $($_.Exception)."
+				Stop-Function -Message "Restore failed: $($_.Exception)."
```

Comment:
Thanks, Claudio! The ErrorRecord actually outputs the text you're looking for. Try this instead `Stop-Function -Message "Restore failed" -ErrorRecord $_ -Target $whateveristhetarget` 

but replace target with whatever you need to. unsure why? Take a look at  `Get-DbatoolsLog | OGV`

Reply:
Thanks @potatoqualitee ! done.